scripts/translation/lib/GitLogParser.php

```diff
@@ -26,10 +26,13 @@ static function parseInto( string $lang , RevcheckFileList & $list )
         $cwd = getcwd();
         chdir( $lang );
         $fp = popen( "git log --name-only" , "r" );
+        chdir( $cwd );
+
```

Comment:
Aside, shouldn't this use `git -C` instead of using  `chdir()`, see: https://git-scm.com/docs/git#Documentation/git.txt--Cltpathgt

Reply:
That's is nice. I will change and test this, in all places, in a separated PR about robust log parsing.